website/tutorial/running-an-ai-application.md

```diff
@@ -0,0 +1,136 @@
+---
+sidebar_position: 2
+title: Running an AI application
+description: Running an AI application using the UI.
+keywords: [podman desktop, podman, AI application]
+tags: [podman-desktop, running-an-ai-application]
+---
+
+# Running an AI application
+
+This tutorial covers the following end-to-end tasks required to run and validate an Artificial Intelligence (AI) application on your local machine:
+
+- Installing the extension
+- Downloading a model
+- Creating a service
+- Optional: Creating a playground
+- Running a recipe
+- Verifying the AI application
+
+Before going through this tutorial, familiarize yourself with the usage of the following sections on the AI Lab page:
+
+![AI lab sections](img/ai-lab-sections.png)
+
+1. **Catalog**: Helps in downloading or importing a model.
+2. **Services**: Helps in creating an inference server that runs in a container to serve a large language model (LLM) using an OpenAI-compatible API. After creating a service, you can access its details and generate its client code in any programming languages to integrate your application.
+3. **Playgrounds**: Helps in experimenting with model parameters and accuracy to identify the best model for your use case. Each playground denotes a customized environment to experiment with a model.
+4. **Recipes Catalog**: Helps in accessing a catalog of recipes, such as chatbot, audio-to-text, code generation, and summarizer. Each recipe denotes an AI application catering to a specific use case. You can use a recipe to start an AI application. You can also clone a recipe to your local machine for editing purposes.
+5. **Running**: Helps in accessing all running AI applications.
+
+This tutorial focuses on running a ChatBot AI application.
+
+## Before you begin
+
+- [Installed Podman Desktop application](/docs/installation).
+- [A running Podman machine](/docs/podman/creating-a-podman-machine) with at least 6 GB memory. If the Podman machine does not have enough memory, you get a notification to update its memory configuration while downloading a model or running a recipe:
+  ![notification to update podman machine configuration](img/update-machine-config.png)
+- A user with developer role.
+
+## Installing the extension
+
+1. Go to **extensions > Catalog**.
+2. Click the **Install** icon to install the Podman AI Lab extension.
+   ![install the extension](img/downloading-ai-lab-ext.png)
+3. Select the **Installed** tab to check the extension is active and a new AI Lab component is added in the left navigation pane.
+   ![extension enabled](img/active-ai-lab-ext-with-ai-lab-icon.png)
+
+## Downloading a model
+
+1. Click **AI Lab** in the left navigation pane.
+2. In the Catalog section, click the **Download** icon corresponding to the model you want to use.
+   ![extension enabled](img/download-a-model.png)
+
+3. View the downloaded model on the same page.
```

Comment:
You could notice that it will be displayed this way only after the download completes